sql/core/src/test/scala/org/apache/spark/sql/FileBasedDataSourceSuite.scala

```diff
@@ -205,6 +205,34 @@ class FileBasedDataSourceSuite extends QueryTest with SharedSQLContext with Befo
     }
   }
 
+  allFileBasedDataSources.foreach { format =>
```

Comment:
@kiszk Add UT in FileBasedDataSourceSuite as we discuss in https://github.com/apache/spark/pull/21618#discussion_r216156188.